### PR TITLE
Support terminal title

### PR DESCRIPTION
**Pull request by gpotter2**
Try to use IPython builtin + on Windows, reset it each time a program reset it (wlanhelper + powershell do reset it)

image

### CODE REVIEW

File: scapy/arch/windows/__init__.py [hunk 1]
```diff
@@ -76,6 +76,11 @@ def _encapsulate_admin(cmd):
     # rights, which will execute the command
     return "Start-Process PowerShell -windowstyle hidden -Wait -Verb RunAs -ArgumentList '-command &{%s}'" % cmd
 
+def _windows_title(title=None):
```
Comment (guedou) [inline scapy/arch/windows/__init__.py]: docstring please.

**Comment by guedou** [comment]
Is there a way to mock this PR?

**Comment by gpotter2** [comment]
Don't think so. This PR is purely esthetic

File: scapy/arch/windows/__init__.py (hunk 1 above)
Comment (guedou) [reply scapy/arch/windows/__init__.py]: PING =)

Comment (gpotter2) [reply scapy/arch/windows/__init__.py]: Skiped that one :) updated

**Pull request merged**